File: rules-interpreter/src/main/java/com/github/iamrenny/ruleflow/evaluators/ParenthesisContextEvaluator.java
package com.github.iamrenny.ruleflow.evaluators;

import com.github.iamrenny.ruleflow.RuleFlowLanguageParser.ParenthesisContext;
import com.github.iamrenny.ruleflow.visitors.Visitor;

public class ParenthesisContextEvaluator implements ContextEvaluator<ParenthesisContext> {

    @Override
    public Object evaluate(ParenthesisContext ctx, Visitor visitor) {
        return visitor.visit(ctx.expr());
    }
}

File: rules-interpreter/src/main/kotlin/com/rappi/analang/evaluators/ParenthesisContextEvaluator.kt
package com.github.iamrenny.rulesflow.evaluators

import com.rappi.fraud.analang.ANAParser.ParenthesisContext
import com.github.iamrenny.rulesflow.visitors.Visitor

class ParenthesisContextEvaluator : ContextEvaluator<ParenthesisContext> {

    override fun evaluate(ctx: ParenthesisContext, visitor: Visitor): Any {
        return visitor.visit(ctx.expr())
    }
}